stress-ng.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: stress-ng
+  version: 0.16.05
+  epoch: 0
+  description: "stress-ng will stress test a computer system in various selectable ways"
+  copyright:
+    - license: GPL-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - keyutils-dev
+      - kmod-dev
+      - libaio-dev
+      - libatomic
+      - libattr1
+      - libbsd-dev
+      - libcap-dev
+      - libgcrypt-dev
+      - libjpeg-dev
```

Comment:
We don't have `libjpeg-dev`. I don't know anything, is https://github.com/wolfi-dev/os/blob/main/libjpeg-turbo.yaml a suitable replacement?

Reply:
Hmm, something is off, but I can't tell what it is.

There's a package here: https://packages.wolfi.dev/os/aarch64/libjpeg-dev-2.1.91-r3.apk and it shows in the index too?

Here's how it looks in the sdk image:
```
[sdk] ❯ apk add -s libjpeg-dev
WARNING: opening ./../../packages: No such file or directory
(1/2) Installing libjpeg-turbo (3.0.0-r0)
(2/2) Installing libjpeg-dev (2.1.91-r3)
OK: 875 MiB in 51 packages
```

Building in the sdk container with this `stress-ng.yaml` works, and I can run the following, which uses the library:

```
[sdk] ❯ stress-ng --jpeg 1 --cpu 1 --timeout 10
stress-ng: info:  [7982] setting to a 10 secs run per stressor
stress-ng: info:  [7982] dispatching hogs: 1 jpeg, 1 cpu
stress-ng: info:  [7982] skipped: 0
stress-ng: info:  [7982] passed: 2: jpeg (1) cpu (1)
stress-ng: info:  [7982] failed: 0
stress-ng: info:  [7982] metrics untrustworthy: 0
stress-ng: info:  [7982] successful run completed in 10.00 secs
```

Anything else I can do to troubleshoot?

Reply:
Oh, yep, that answers my question. libjpeg-turbo also builds that libjpeg-dev package, so you're good to go

https://apk.dag.dev/https/packages.wolfi.dev/os/x86_64/libjpeg-dev-2.1.91-r3.apk@sha1:478dc66cb959efee4efb7b0eb3f33f73535ce926/.PKGINFO says `origin = libjpeg-turbo` and links to the commit where it was built from, a few months ago: https://github.com/wolfi-dev/os/blob/c4884554ebfac762b91feb7ef1b17912fe9c5997/libjpeg-turbo.yaml#L45

However, after that, https://github.com/wolfi-dev/os/commit/88960572c4c2f73e9b64592682e2a6378afff7c1 renamed the subpackage to `libjpeg-turbo-dev`, which is the one currently getting updates (it's on 3.0.0 now)

So `libjpeg-dev` exists but it's old and not getting updates. `libjpeg-turbo-dev` is the new hotness, and will remain the new hotness going forward. 😄

Reply:
Mystery solved, thanks for tracking that down! I updated the PR to use `libjpeg-turbo-dev`, let's see what CI says after this update.